Documentation/gettingstarted/external-workloads.rst

```diff
@@ -43,190 +45,109 @@ Prerequisites
+  resolution service by a stand-alone ``/etc/resolv.conf``, or via
+  systemd (e.g., Ubuntu).
+
+* So far this functionality is only tested with the vxlan tunneling
+  datapath mode (default for most installations).
 
 Prepare your cluster
 ####################
 
 Enable support for external workloads
 =====================================
 
-.. include:: k8s-install-download-release.rst
-
 Your cluster must be configured with support for external workloads
-enabled. This can also be done by passing ``--set
-externalWorkloads.enabled=true`` to ``helm install`` when installing
-or updating Cilium:
-
-.. parsed-literal::
-
-    helm install cilium |CHART_RELEASE|         \\
-      --namespace kube-system                   \\
-      --set externalWorkloads.enabled=true
-
-This will add a deployment for ``clustermesh-apiserver`` into your
-cluster, as well as the related cluster resources, such as TLS
-secrets.
-
-.. _gs_external_workloads_tls_configuration:
-
-TLS configuration
-=================
-
-By default TLS secrets are created by helm. This has the downside that
-each time you run ``helm`` the TLS secrets will be re-created, forcing
-each external workload to be reconfigured to be able to connect to
-your cluster. There are two ways to get around this. You can
-enable a cluster job to create TLS secrets instead. This way the CA
-cert secrets are reused as long as they are not removed from your
-cluster:
-
-.. parsed-literal::
-
-    helm install cilium |CHART_RELEASE|                     \\
-      --namespace kube-system                               \\
-      --set externalWorkloads.enabled=true                  \\
-      --set clustermesh.apiserver.tls.auto.method=cronJob
-
-Alternatively, you can use your own CA certs. You can either create the
-``clustermesh-apiserver-ca-cert`` secret in your Cilium install
-namespace (e.g., ``kube-system``) yourself, or pass the CA cert and
-key to helm:
+enabled. This can be done with the cilium CLI tool by issuing ``cilium
+clustermesh enable`` after ``cilium install``:
 
 .. parsed-literal::
 
-    helm install cilium |CHART_RELEASE|                     \\
-      --namespace kube-system                               \\
-      --set externalWorkloads.enabled=true                  \\
-      --set clustermesh.apiserver.tls.auto.method=cronJob   \\
-      --set clustermesh.apiserver.tls.ca.cert="<< base64 encoded CA certificate >>"   \\
-      --set clustermesh.apiserver.tls.ca.key="<< base64 encoded CA key >>"
-
-You can also allow the cronJob create the secrets using the first approach above and then save the
-generated CA certs to be used with the second approach above:
-
-.. code:: bash
-
-    kubectl -n kube-system get secret clustermesh-apiserver-ca-cert -o yaml > clustermesh-apiserver-ca-cert.yaml
-   
-.. _gs_external_workloads_expose_clustermesh_apiserver:
-
-Expose cluster to external workloads
-====================================
+    cilium install --config tunnel=vxlan
```

Comment:
Nit: The directive above this block no longer needs to be a `.. parsed-literal::`, could you please switch to `.. code-block:: shell-session`?

Reply:
Thanks for the guidance; changed all directives on shell commands/output to `.. code-block:: shell-session`